debezium-core/src/main/java/io/debezium/relational/RelationalChangeRecordEmitter.java

```diff
@@ -134,7 +139,8 @@ protected void emitDeleteRecord(Receiver receiver, TableSchema tableSchema) thro
             return;
         }
 
-        Struct envelope = tableSchema.getEnvelopeSchema().delete(oldValue, getOffset().getSourceInfo(), getClock().currentTimeAsInstant());
+        Instant instant = getInstantFrom(getOffset());
```

Comment:
I'm not entirely sure the relational behind this change either, @AndreyIg can you or Mallik share your reasonings?

Reply:
This could be suspicious but I couldn't find the place where to add real CDC event time. There are three time measures in SQL server world
  1.  Record created in DB (which will go part of source record payload and need not worried about it)
  2.  What time it arrived to CDC  tables (few millis off from real source record created time)
  3.  When connector read this event (when connector retrieved this event to process. Usually always System.currentTimeMillis())

But I want to measure second one meaning when connector is down for 30min and restarted, we can get 1 & 3 but not 2. I realized instead of System.currentTimeMillis() better to measure based on CDC created time.

Reply:
Ok, I see. As you say:

1. is the top-level ts_ms in change events. That why that change above should be reverted.
3. is source.ts_ms

There could be a new field for 2. under source for SQL Server? How can that info be obtained, though?